PanoptesNetClient/PanoptesNetClient/ApiClient.cs

```diff
@@ -35,14 +37,15 @@ public ApiClient(HttpClient client = null)
                 ConfigClient();
             }
         }
+        #endregion
 
         /// <summary>
         /// Make a GET request for a single resource using IRequest
         /// </summary>
-        #region Generic GET using IRequest
-        public async Task<T> Get<T>(IRequest request)
+        #region Generic GET of single resource
+        public async Task<T> Get<T>(IRequest request) where T:IResource
```

Comment:
It may be a good idea to make this function `protected` instead of `public`. `protected` will  make it so classes that inherit from `ApiClient` can access the function, but other classes can't. 

This will force the consumer to use the `ApiClient` subclasses.